packages/odyssey-react/src/components/Banner/Banner.test.tsx

```diff
@@ -67,7 +67,7 @@ describe("Banner", () => {
       </Banner>
     );
 
-    expect(getByRole(role)).toHaveClass(styles.isDismissed);
+    expect(getByRole(role, { hidden: true })).toHaveClass(styles.isDismissed);
```

Comment:
This is interesting: now that styles are being injected into JSDOM during tests via withStyles, since the banner is hidden via that CSS it is no longer accessible! Have to use an optional param here for the testing-library util to find the role otherwise this test fails